common/lib/xmodule/xmodule/tests/test_video.py

```diff
@@ -1073,3 +1073,13 @@ def test_no_transcript_validation_message(self, xml_transcripts, expected_valida
         descriptor = instantiate_descriptor(data=xml_data_transcripts)
         validation = descriptor.validate()
         self.assert_validation_message(validation, expected_validation_msg)
+
+    def test_video_transcript_none(self):
+        """
+        Test video with transcripts being overridden to None.
```

Comment:
Keeping in mind your new implementation this doc string is no more valid. Now `transcripts` would never be `None`.